### PR TITLE
Fixes #26, #28 - Updating dependencies

### LINKED ISSUE #26: An in-range update of lerna is breaking the build 🚨
## The devDependency [lerna](https://github.com/lerna/lerna) was updated from `3.7.0` to `3.7.1`.

🚨 [View failing branch](https://github.com/zone/frontend/compare/develop...zone:greenkeeper%2Flerna-3.7.1).

This version is **covered** by your **current version range** and after updating it in your project **the build failed**.

lerna is a devDependency of this project. It **might not break your production code or affect downstream projects**, but probably breaks your build or test tools, which may **prevent deploying or publishing**.

<details>
<summary>Status Details</summary>

- ❌ **continuous-integration/travis-ci/push:** The Travis CI build failed ([Details](https://travis-ci.org/zone/frontend/builds/470306505?utm_source=github_status&utm_medium=notification)).
</details>

---

<details>
<summary>Release Notes for v3.7.1</summary>

<h3>Bug Fixes</h3>
<ul>
<li><strong>bootstrap:</strong> Pulse progress bar during execution (<a href="https://urls.greenkeeper.io/lerna/lerna/commit/b38a151">b38a151</a>)</li>
<li><strong>clean:</strong> Pulse progress bar during execution (<a href="https://urls.greenkeeper.io/lerna/lerna/commit/f1202de">f1202de</a>)</li>
<li><strong>command:</strong> Enable progress from top-level log object (<a href="https://urls.greenkeeper.io/lerna/lerna/commit/95e88f0">95e88f0</a>)</li>
<li><strong>import:</strong> Pulse progress bar during execution (<a href="https://urls.greenkeeper.io/lerna/lerna/commit/b552e22">b552e22</a>)</li>
<li><strong>npm-dist-tag:</strong> Accept <code>opts.log</code>, defaulting to <code>libnpm/log</code> (<a href="https://urls.greenkeeper.io/lerna/lerna/commit/97edc7e">97edc7e</a>)</li>
<li><strong>npm-publish:</strong> Accept <code>opts.log</code>, defaulting to <code>libnpm/log</code> (<a href="https://urls.greenkeeper.io/lerna/lerna/commit/a1d61f6">a1d61f6</a>)</li>
<li><strong>pack-directory:</strong> Accept <code>opts.log</code>, defaulting to <code>libnpm/log</code> (<a href="https://urls.greenkeeper.io/lerna/lerna/commit/d099d13">d099d13</a>)</li>
<li><strong>publish:</strong> Do not pass <code>this.logger</code> into conf, it does not respect <code>log.level</code> (<a href="https://urls.greenkeeper.io/lerna/lerna/commit/9bcd503">9bcd503</a>)</li>
<li><strong>publish:</strong> Pulse progress bar during execution (<a href="https://urls.greenkeeper.io/lerna/lerna/commit/49b8771">49b8771</a>)</li>
<li>Add <code>@lerna/pulse-till-done</code> utility (<a href="https://urls.greenkeeper.io/lerna/lerna/commit/3359c63">3359c63</a>)</li>
<li><strong>run-lifecycle:</strong> Accept <code>opts.log</code>, defaulting to <code>libnpm/log</code> (<a href="https://urls.greenkeeper.io/lerna/lerna/commit/dde588a">dde588a</a>)</li>
<li><strong>run-lifecycle:</strong> Do not execute on packages that lack the target script, avoiding spurious logs (<a href="https://urls.greenkeeper.io/lerna/lerna/commit/c0ad316">c0ad316</a>)</li>
</ul>
</details>

<details>
<summary>Commits</summary>
<p>The new version differs by 13 commits.</p>
<ul>
<li><a href="https://urls.greenkeeper.io/lerna/lerna/commit/284aa4747dad26977790d6730ab2f640eb099cb4"><code>284aa47</code></a> <code>chore(release): publish v3.7.1</code></li>
<li><a href="https://urls.greenkeeper.io/lerna/lerna/commit/9bcd50336656c157e7973f46ffc121f754874e8e"><code>9bcd503</code></a> <code>fix(publish): Do not pass this.logger into conf, it does not respect log.level</code></li>
<li><a href="https://urls.greenkeeper.io/lerna/lerna/commit/c0ad316cb66e558e16eed7dcd6d38c2c7209a9b6"><code>c0ad316</code></a> <code>fix(run-lifecycle): Do not execute on packages that lack the target script, avoiding spurious logs</code></li>
<li><a href="https://urls.greenkeeper.io/lerna/lerna/commit/dde588a8525e054747673df9daebaebda9a855ca"><code>dde588a</code></a> <code>fix(run-lifecycle): Accept opts.log, defaulting to libnpm/log</code></li>
<li><a href="https://urls.greenkeeper.io/lerna/lerna/commit/a1d61f61da6042d2c6ab1537c49dc6a6711ed243"><code>a1d61f6</code></a> <code>fix(npm-publish): Accept opts.log, defaulting to libnpm/log</code></li>
<li><a href="https://urls.greenkeeper.io/lerna/lerna/commit/97edc7eea8ef5f6450a7cbaeb6baba792b0a4f2e"><code>97edc7e</code></a> <code>fix(npm-dist-tag): Accept opts.log, defaulting to libnpm/log</code></li>
<li><a href="https://urls.greenkeeper.io/lerna/lerna/commit/d099d132372949d4599c3ffaa1e401ffc47611f9"><code>d099d13</code></a> <code>fix(pack-directory): Accept opts.log, defaulting to libnpm/log</code></li>
<li><a href="https://urls.greenkeeper.io/lerna/lerna/commit/49b877131fe5422a7930914cad4e688eb7bcd04b"><code>49b8771</code></a> <code>fix(publish): Pulse progress bar during execution</code></li>
<li><a href="https://urls.greenkeeper.io/lerna/lerna/commit/b552e2280a00e82a6a68ec9244b21b6aedadd6ce"><code>b552e22</code></a> <code>fix(import): Pulse progress bar during execution</code></li>
<li><a href="https://urls.greenkeeper.io/lerna/lerna/commit/f1202de58feb01940242de5ab810ac39ba795d04"><code>f1202de</code></a> <code>fix(clean): Pulse progress bar during execution</code></li>
<li><a href="https://urls.greenkeeper.io/lerna/lerna/commit/b38a151bb2b32fcd1db3071ff5afe0b01089bbef"><code>b38a151</code></a> <code>fix(bootstrap): Pulse progress bar during execution</code></li>
<li><a href="https://urls.greenkeeper.io/lerna/lerna/commit/3359c637d12ebf7a5ec228395375794e61972c65"><code>3359c63</code></a> <code>fix: Add pulse-till-done utility</code></li>
<li><a href="https://urls.greenkeeper.io/lerna/lerna/commit/95e88f026d20a0f3d57d4f63c02987e698949528"><code>95e88f0</code></a> <code>fix(command): Enable progress from top-level log object</code></li>
</ul>
<p>See the <a href="https://urls.greenkeeper.io/lerna/lerna/compare/89b53ff2b561e69fad3a3e8671d7dcd0f1c09c3e...284aa4747dad26977790d6730ab2f640eb099cb4">full diff</a></p>
</details>

<details>
<summary>FAQ and help</summary>

There is a collection of [frequently asked questions](https://greenkeeper.io/faq.html). If those don’t help, you can always [ask the humans behind Greenkeeper](https://github.com/greenkeeperio/greenkeeper/issues/new).
</details>

---

Your [Greenkeeper](https://greenkeeper.io) Bot :palm_tree:

### PR DESCRIPTION
**Pull request by simmo**
**What does this PR cover?**
Bugfix

**Breaking change?**
Minor

**Are all the tests are passing?**
Yes

**Other information:**
This has a minor change for the vue config, that removes a rule.

### CODE REVIEW

**Comment by simmo** [comment]
Worth saying the fix from #29 should stop these kind of Greenkeeper issues from appearing.

File: packages/eslint-config-vue/__snapshots__/index.test.js.snap [hunk 1]
```diff
@@ -2097,7 +2097,6 @@ Object {
     "vue/attribute-hyphenation": "error",
     "vue/attributes-order": "warn",
     "vue/comment-directive": "error",
-    "vue/component-name-in-template-casing": "warn",
```
Comment (simmo) [inline packages/eslint-config-vue/__snapshots__/index.test.js.snap]: Updating vue's recommended config to 5.1 removes this rule.
More info on it here... https://vuejs.github.io/eslint-plugin-vue/rules/component-name-in-template-casing.html

If is still available but has been removed from 'recommended' to 'uncategorised'.

Comment (sburrows-zone) [reply packages/eslint-config-vue/__snapshots__/index.test.js.snap]: fyi reasons for change: https://github.com/vuejs/eslint-plugin-vue/pull/703

Comment (simmo) [reply packages/eslint-config-vue/__snapshots__/index.test.js.snap]: Ah nice one, thanks.

**Pull request merged**